lms/djangoapps/certificates/api.py

```diff
@@ -484,9 +484,9 @@ def get_active_web_certificate(course, is_preview_mode=None):
     return None
 
 
-def get_certificate_template(course_key, mode):
+def get_certificate_template(course_key, mode, language):  # pylint: disable=unused-argument
```

Comment:
Why are we passing an unused argument?

Reply:
It will be used in another ticket that Turc is implementing. I added it to the contract of the method now so that I could include send the language when calling this method from a file that is more under the jurisdiction of my ticket.